erpnext/agriculture/doctype/plant_batch/plant_batch.js

```diff
@@ -45,8 +51,42 @@ frappe.ui.form.on('Plant Batch', {
 					frm: frm
 				});
 			}, __("Create"));
+			frm.add_custom_button(__("Destroy Plant Batch"), () => {
```

Comment:
The context is already there

```suggestion
			frm.add_custom_button(__("Destroy"), () => {
```